### PR TITLE
[NUI] Fix Position type converter error when Position get 2 parameters in XAML

### PR DESCRIPTION
**Pull request by elishateng**
### Description of Change ###

1. Fix Position type converter error when Position get 2 parameters in XAML

### API Changes ###

 - ACR:

### CODE REVIEW

File: src/Tizen.NUI/src/internal/XamlBinding/PositionTypeConverter.cs [hunk 1]
```diff
@@ -59,6 +59,12 @@ public override object ConvertFromInvariantString(string value)
                     int z = (int)GraphicsTypeManager.Instance.ConvertScriptToPixel(parts[2].Trim());
                     return new Position(x, y, z);
                 }
+                else if (parts.Length == 2)
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/internal/XamlBinding/PositionTypeConverter.cs:62]: Position and Size gets float typed parameter. and as I know xaml support float type. (maybe we can use Single class)
but this looks like converted and int type. 
how do you think about?

File: src/Tizen.NUI/src/public/XamlBinding/SizeTypeConverter.cs [hunk 2]
```diff
@@ -28,6 +28,12 @@ public override object ConvertFromInvariantString(string value)
                     int z = (int)GraphicsTypeManager.Instance.ConvertScriptToPixel(parts[2].Trim());
                     return new Size(x, y, z);
                 }
+                else if (parts.Length == 2)
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/public/XamlBinding/SizeTypeConverter.cs:31]: same as above

File: src/Tizen.NUI/src/internal/XamlBinding/PositionTypeConverter.cs (hunk 1 above)
Comment (elishateng) [reply src/Tizen.NUI/src/internal/XamlBinding/PositionTypeConverter.cs:62]: Hello, Mr. SSong, You're right. Thank you for your remind.

File: src/Tizen.NUI/src/public/XamlBinding/SizeTypeConverter.cs (hunk 2 above)
Comment (elishateng) [reply src/Tizen.NUI/src/public/XamlBinding/SizeTypeConverter.cs:31]: Hello, Mr. SSong, You're right. Thank you for your remind.

**Pull request merged**